remote-data-update.go

```diff
@@ -36,18 +37,33 @@ func main() {
 		if err != nil {
 			log.Fatal(v, err)
 		}
-		r, err := http.Get(v)
+
+		req, err := http.NewRequest(http.MethodGet, v, nil)
+		if err != nil {
+			log.Fatal(v, err)
+		}
+
+		// Twitter only returns Open Graph data for specific user agents
+		// Here we pretend we are the Facebook crwaler
+		if strings.Contains(v, "://twitter.com") {
+			req.Header.Set("User-Agent", "facebookexternalhit/1.1 (+http://www.facebook.com/externalhit_uatext.php)")
```

Comment:
Please use `DiscourseBot/1.0`, if possible. Reasons outlined [here](https://github.com/Doist/Issues/issues/2456#issuecomment-849179829). 🙌 

We can also include a link to the discussion where Twitter recommends this in the comment, for future reference.

Reply:
Same below. 👍